app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -6,6 +6,18 @@
     color: $color-select;
 }
 
+.download-dropdown-content {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    border: 1px solid $color-border-gray;
+    width: max-content;
+}
+
+.downlaod-link {
```

Comment:
At least I copy-pasted this typo throughout, so it's applying the correct styles 😆